ophyd/pseudopos.py

```diff
@@ -327,6 +331,29 @@ class PseudoPositioner(Device, SoftPositioner):
     timeout : float, optional
         The default timeout to use for motion requests, in seconds.
     '''
+    class __add_sub_mixin:
+        """Helper mix-in to make RealPosition and PseudoPosition mathable
+
+        .. warning ::
+
+           Do not use this outside of this purpose.
+
+           No really, don't
+        """
+        __slots__ = ()
+
+        def __add__(self, other):
+            if not isinstance(other, type(self)):
+                raise NotImplemented
+            return type(self)(*(s + o for s, o in
+                                zip(self, other)))
+
+        def __sub__(self, other):
+            if not isinstance(other, type(self)):
+                raise NotImplemented
```

Comment:
Refer to https://docs.python.org/3.6/library/constants.html#NotImplemented

I believe the proper way is to either return `NotImplemented` or raise `NotImplementedError` - they aren't interchangeable.

Reply:
Got that I wanted `NotImplemented` but not that I wanted to return it :disappointed: